src/main/resources/l10n/JabRef_da.properties

```diff
@@ -2302,3 +2302,5 @@ Select_first_entry=
 Select_last_entry=
 
 Search_in_all_open_databases=
+
+Invalid_ISBN\:_'%0'.=
```

Comment:
Ah, yes, we have Invalid_DOI already. It would make sense to have sth like Invalid_%0 which we can reuse for all id fetchers. (e.g. doi/isbn) so we